src/components/Card/Card.scss

```
@@ -0,0 +1,74 @@
+.card {
+  max-width: 306px;
+  height: 502px;
+  
+  * + * {
+    padding-left: 47px;
+    padding-right: 47px;
+  }
+
+  & h1 {
+    color: white;
+    font-size: 32px;
+    margin-top: 15px;
+    margin-bottom: 15px;
+  }
+
+  & p {

```

same as here https://github.com/MarcelloAragoni/mentor/pull/2/files#r844136076

Another point in SCSS is you don't need to put `&` alone at the start of the line:

You use when you want to construct a selector based on the parent selector (`.card`)
```scss
.card {
  &__title { ... }  // <<<< interpolating & in __title
}
```

So you don't want to use it alone:
```suggestion
  p {
```

That's because SCSS already knows that you're inside the `.card`, doing it:

```
.card  {
  .card__title {}
}
```

it's pretty much the same as :
```
.card  {
  &  .card__title {}
}
```